### PR TITLE
Add logging for keep-alive timeout 

### PR DESCRIPTION
**Pull request by nadeshpa**
Add logging for keep-alive timeout.

### CODE REVIEW

File: src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs [hunk 1]
```diff
@@ -932,7 +932,7 @@ static void CheckKeepAlive(object ctx)
             TimeSpan elapsedSinceLastActive = DateTime.UtcNow - self.lastClientActivityTime;
             if (elapsedSinceLastActive > self.keepAliveTimeout)
             {
-                ShutdownOnError(context, string.Empty, new ProtocolGatewayException(ErrorCode.KeepAliveTimedOut, "Keep Alive timed out."));
+                ShutdownOnError(context, string.Empty, new ProtocolGatewayException(ErrorCode.KeepAliveTimedOut, $"Keep Alive timed out {self.keepAliveTimeout}."));
```
Comment (nayato) [inline src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs:935]: $"Keep Alive timed out. Timeout: {self.keepAliveTimeout}"

File: src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs [hunk 2]
```diff
@@ -890,7 +890,7 @@ TimeSpan DeriveKeepAliveTimeout(IChannelHandlerContext context, ConnectPacket pa
         /// <param name="context"><see cref="IChannelHandlerContext" /> instance.</param>
         void CompleteConnect(IChannelHandlerContext context)
         {
-            CommonEventSource.Log.Info("Connection established.", this.ChannelId, this.Id);
+            CommonEventSource.Log.Info($"Connection established. Keep-alive timeout {this.keepAliveTimeout}.", this.ChannelId, this.Id);
```
Comment (nayato) [inline src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs:893]: "... Keep-Alive timeout: {this.keepAliveTimeout}"

**Review by nayato — CHANGES_REQUESTED** [review]

File: src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs (hunk 1 above)
Comment (nadeshpa) [reply src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs]: I will modify the log string

File: src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs (hunk 2 above)
Comment (nadeshpa) [reply src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs]: I will modify the log string

**Review by nayato — APPROVED** [review]

**Pull request merged**